Find a functionally equivalent code:android.media.MediaPlayer.addSubtitleSource:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -27,33 +27,33 @@
             }
             scanner.close();
             mOutOfBandSubtitleTracks.add(track);
-            track.onData(contents, true, /* eos */
+            track.onData(contents.getBytes(), true, /* eos */
             ~0);
             return MEDIA_INFO_EXTERNAL_METADATA_UPDATE;
         }

